commands/util/MigrationCommand.ts

```diff
@@ -33,28 +34,49 @@ export default abstract class MongodbMakeMigration extends BaseCommand {
   @flags.string({ description: 'Database connection to use for the migration' })
   public connection: string;
 
-  protected getMigrationFilePath(file: string) {
-    return join(this.application.appRoot, folder, file);
-  }
+  protected async getMigrationFiles(
+    config: MongodbConnectionConfig,
+  ): Promise<string[]> {
+    const folders =
+      config.migrations && config.migrations.length > 0
```

Comment:
I'm ok with this but it does not match your comment on the PR

You said that specifying the options will still look in the default folder

Reply:
Yep sorry didn't update.
We discussed with Targos about an edge-case where we could want some connections to not execute migrations located in the default folder. So we don't anymore.